Extensions/xkit_preferences.js

```diff
@@ -378,14 +382,36 @@ XKit.extensions.xkit_preferences = new Object({
 			var show_all = XKit.tools.get_setting("xkit_show_feature_updates","true") === "true";
 
 			var m_return = 0;
+			var show_information = JSON.parse(XKit.tools.get_setting("xkit_show_information","true"));
+			var show_extension_updates = JSON.parse(XKit.tools.get_setting("xkit_show_extension_updates","true"));
+			var show_weekly_news = JSON.parse(XKit.tools.get_setting("xkit_show_weekly_news","true"));
+			var show_developer_news = JSON.parse(XKit.tools.get_setting("xkit_show_developers","true"));
+			var show_firefox = true;
```

Comment:
Below this there's a lot of duplicate setters for `show_firefox` and the like. You may be able to do this instead:

```js
var show_firefox = JSON.parse(XKit.tools.get_setting("xkit_show_firefox_news", XKit.browser().firefox);
var show_chrome = JSON.parse(XKit.tools.get_setting("xkit_show_chrome_news", XKit.browser().chrome);
var show_safari = JSON.parse(XKit.tools.get_setting("xkit_show_safari_news", XKit.browser().safari);
```